server/src/main/java/io/druid/query/lookup/MapLookupExtractorFactory.java

```diff
@@ -103,4 +119,29 @@ public int hashCode()
     result = 31 * result + (isOneToOne ? 1 : 0);
     return result;
   }
+
+  public static class MapLookupIntrospection implements LookupIntrospectHandler
+  {
+    final private Map<String, String> map;
+    public MapLookupIntrospection(Map<String, String> map)
+    {
+      this.map = map;
+    }
+
+    @GET
+    @Path("/keys")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getKeys()
+    {
+      return Response.ok(map.keySet().toString()).build();
+    }
+
+    @GET
+    @Path("/values")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getValues()
+    {
+      return Response.ok(map.values().toString()).build();
+    }
+  }
```

Comment:
why not a method to dump whole map instead of 2 which dump keys or values? usually , I would like to see whole relation ship in the case of map lookup.